src/format/command.rs

```diff
@@ -10,8 +12,44 @@ pub fn run(stdin: bool, check: bool, files: Vec<String>) -> Result<(), Error> {
     }
 }
 
-pub fn format_files(_files: Vec<String>, _check: bool) -> Result<(), Error> {
-    todo!()
+pub fn format_files(files: Vec<String>, _check: bool) -> Result<(), Error> {
+    for file_path in files {
+        let path = PathBuf::from_str(&file_path).map_err(|e| Error::FileIO {
+            action: FileIOAction::Open,
+            kind: FileKind::File,
+            path: PathBuf::new(),
+            err: Some(format!("Invalid path: {}\n{}", file_path, e.to_string())),
+        })?;
+
+        let mut file = OpenOptions::new()
+            .write(true)
+            .read(true)
+            .open(path.clone())
+            .expect(&format!("Could not open file: {}", file_path));
+
+        let mut src = String::new();
+        file.read_to_string(&mut src)
+            .expect(&format!("Could not read file: {}", file_path));
+
+        let formatted = crate::format::pretty(src.as_ref()).map_err(|error| Error::Parse {
+            path: PathBuf::from("<standard input>"),
```

Comment:
The error message here says the code is from stanard in, but it's from a file. Let's use the path of the file here to make the error message accurate.

Reply:
Done.  The problem with copy-pasta